### DISCUSSION TITLE
如何使EditAnywhere/EditInstanceOnly生效

### DISCUSSION THREAD

**Discussion by NashXY** [Tencent/puerts#1903] [Q&A]
![image](https://github.com/user-attachments/assets/e5e4c673-f471-4386-9d7a-5116a6645b8a)
在蓝图中勾选Instance Editable或在C++给UPROPERTY配置EditAnywhere或EditInstanceOnly时，可以在场景详细中显示编辑此属性
![image](https://github.com/user-attachments/assets/5393cd7f-4772-4575-be9e-a1a23ceec523)

但是如果在Puerts中给UPROPERTY配置EditAnywhere时，却无法在场景详细面板中显示
![image](https://github.com/user-attachments/assets/336853fc-f419-4e87-be96-b2cdfd768f54)
![image](https://github.com/user-attachments/assets/f3d791b0-ef79-4810-b637-9007cf7c6361)

要如何能解决这个问题

**Comment by chexiongsheng** [comment]
试试
@UE.edit_on_instance()

**Reply by NashXY** [reply]
这好像不管用，我在代码中加上了
`    @UE.edit_on_instance()
    public TestValue: boolean;`
    
![image](https://github.com/user-attachments/assets/37d3609c-5936-49d8-a6e6-d5088d51c5bc)
当在场景中的Actor详细面板依然没有出现对应属性
![image](https://github.com/user-attachments/assets/e29f19b8-3af7-4c9d-9aeb-7ccdaefe3b1a)

我puerts插件的版本是11月7号的
![image](https://github.com/user-attachments/assets/901962a3-a606-4f1a-9eb2-165380fde1f6)

**Reply by NashXY** [reply]
@chexiongsheng
